week1/homework/src/server.js

```diff
@@ -8,8 +8,37 @@ const http = require('http');
 function createServer(port) {
   let state = 10;
 
+  function displayResponse(response, state) {
+    response.setHeader('Content-Type', 'application/json');
+    response.write(JSON.stringify(state));
+    response.end();
+  }
+
   const server = http.createServer((request, response) => {
-    // TODO: Write your homework code here
+    switch (request.url) {
+      case '/state':
+        displayResponse(response, { state });
+        break;
+
+      case '/add':
+        state++;
+        displayResponse(response, { state });
```

Comment:
honestly had to look up how `{ state }` works. I like it much more than  `{ 'state': state } ` Very good 👍